mesmer/core/localized_covariance.py

```diff
@@ -0,0 +1,227 @@
+import numpy as np
+from scipy.stats import multivariate_normal
+
+
+def _adjust_ecov_ar1_np(covariance, ar_coefs):
+    """
+    adjust localized empirical covariance matrix for autoregressive process of order 1
+
+    Parameters
+    ----------
+    covariance : 2D np.array
+        Empirical covariance matrix.
+    ar_coefs : 1D np.array
+        The coefficients of the autoregressive process of order 1.
```

Comment:
```suggestion
        The coefficients of the autoregressive process of order 1.
        Must have length equal to the size of `covariance` (`covariance` is square so it doesn't matter which dimension you use)
```

Could also add this assertion in the function if I have understood this correctly (I may not have)

Reply:
:+1: